index.js

```diff
@@ -395,7 +398,8 @@ app.get("/reset", async (req, res) => {
         lname text NOT NULL,
         email text NOT NULL,
         password text NOT NULL,
-        signature text
+        signature text,
+        administrator bool
```

Comment:
Adding `NOT NULL` will avoid the three state problem since `administrator` can be NULL.
```
The boolean type can have several states: "true", "false", and a third state, "unknown", which is represented by the SQL null value.
```

Reply:
Adding a `DEFAULT` value wouldn't hurt either.

Reply:
yes, I think for now it is fine for it to be optional.    we can decide when we add the logic to check this state whether it makes sense to always set it (by default I think it will be set to either false or null and only set if you want the user to be an administrator), for now I did it this way to avoid breaking any other existing scenarios.

Reply:
I will be checking this flag to determine which part of the application to send a person. I think this will work for me:
`administrator bool NOT NULL DEFAULT FALSE`

Reply:
done